doc/cprover-manual/properties.md

```diff
@@ -132,9 +132,10 @@ The goto-instrument program supports these checks:
 As all of these checks apply across the entire input program, we may wish to
 disable them for selected statements in the program. For example, unsigned
 overflows can be expected and acceptable in certain instructions even when
-elsewhere we do not expect them. To selectively disable automatically generated
-properties use `#pragma CPROVER check disable "<name_of_check>"`, which remains
-in effect until a `#pragma CPROVER check pop` (to re-enable all properties
+elsewhere we do not expect them. The latest development version of CBMC (version 5.12)
```

Comment:
5.12 has been released and is available from https://github.com/diffblue/cbmc/releases, but @kroening hasn't updated the cprover website yet.

Reply:
I don't think we should be referring to the "latest" version in documentation, we should instead say: "CBMC 5.12 (or any later version) supports ..."